.github/workflows/ci.yml

```diff
@@ -69,6 +69,46 @@ jobs:
       - name: Verify successful installation
         run: $HOME/bin/dotenv-linter -v
 
+  install_specific_versions:
+    strategy:
+      matrix:
+        version: [3.2.0, 2.2.1, 2.0.0, 1.1.1]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        include:
+          - shell: sh
+          - os: windows-latest
+            shell: bash # it uses Git Bash
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup dotenv-linter version ${{ matrix.version }}
+        shell: ${{ matrix.shell }}
+        run: |
+          mkdir -p $HOME/bin
+          sh "${GITHUB_WORKSPACE}/install.sh" -b $HOME/bin v${{ matrix.version }}
+      - name: Verify successful installation
+        shell: ${{ matrix.shell }}
+        run: test "$($HOME/bin/dotenv-linter -v)" == "dotenv-linter ${{ matrix.version }}"
```

Comment:
I am new to workflows. Is it ok to use the `test` command like that there?

Reply:
I don't see anything wrong with that 🙂